eng/SourceBuild.props

```diff
@@ -31,10 +31,11 @@
       <InnerBuildArgs>$(InnerBuildArgs) --nodereuse false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --warnAsError false</InnerBuildArgs>
       <InnerBuildArgs>$(InnerBuildArgs) --outputrid $(TargetRid)</InnerBuildArgs>
-      <!-- RuntimeOS is the build host rid OS. -->
-      <InnerBuildArgs>$(InnerBuildArgs) /p:RuntimeOS=$(RuntimeOS)</InnerBuildArgs>
+      <!-- PackageOS and ToolsOS control the rids of prebuilts consumed by the build.
+           They are set to RuntimeOS so they match with the build SDK rid. -->
+      <InnerBuildArgs Condition="'$(RuntimeOS)' != ''">$(InnerBuildArgs) /p:PackageOS=$(RuntimeOS) /p:ToolsOS=$(RuntimeOS)</InnerBuildArgs>
```

Comment:
Who provides RuntimeOS in source build?

Reply:
It gets set here: https://github.com/dotnet/dotnet/blob/3b38fd69120ec5cac0433923225a8d6f1d40c0bf/repo-projects/runtime.proj#L16.

Reply:
Is that still the right property to set or do we want to change that?